src/test/scala/it/pagopa/interop/agreementmanagement/SpecHelper.scala

```diff
@@ -4,9 +4,10 @@ import akka.actor
 import akka.http.scaladsl.marshalling.Marshal
 import akka.http.scaladsl.model.{HttpMethods, HttpResponse, MessageEntity}
 import akka.http.scaladsl.unmarshalling.{Unmarshal, Unmarshaller}
+import cats.implicits._
```

Comment:
```suggestion
import cats.syntax.all._
```
Using `cats.implicits` is deprecated